kbase-extension/scss/partials/_bulkImportCell.scss

```diff
@@ -0,0 +1,150 @@
+/*
+    bulk import cell
+*/
+
+.kb-bulk-import {
+  &__base-node {
+    // everything lives under this
+  }
+
+  &__layout_container {
+    display: flex;
+    align-items: stretch;
+  }
+
+  /* kb-bulk-import__prompt */
+  &__prompt {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  /* kb-bulk-import__prompt_status */
+  &__prompt_status {
+  }
+
+  /* kb-bulk-import__body */
+  &__body {
+    display: flex;
+    width: 100%;
+    flex: 1;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  /* kb-bulk-import__widget_container */
+  &__widget_container {
+    display: block;
+    width: 100%;
+  }
+
+  /* kb-bulk-import__cell_container */
+  &__cell_container {
+  }
+
+  /* kb-bulk-import__tab_pane */
+  &__tab_pane {
+  }
+
+  /* kb-bulk-import__tab_pane_widget */
+  &__tab_pane_widget {
+  }
+
+  /* kb-bulk-import-action-button */
```

Comment:
Where possible, I used the naming schema

`${cssCellType}-${file/componentName}__$element(--$modifier)?`

where `cssCellType` is `kb-bulk-import`, and `file/componentName` is derived from the component or the name of the file, i.e. `-action-button` for `actionButton.js`, `-tabs` for `cellTabs.js`, `-control-panel` for `cellControlPanel.js`.

Reply:
sensible naming scheme 👌  is this documented anywhere?